src/components/Player/index.tsx

```diff
@@ -80,6 +101,30 @@ const PreviewContainer = styled.div`
   }
 `
 
+const Model3dContainer: React.FC<{ renderer?: string }> = ({ renderer }) => {
+  const [isLoading, setLoading] = useState(true)
```

Comment:
names are not consistent
`[loading, setLoading]` or `[isLoading, setIsLoading]`